README.md

```diff
@@ -13,18 +13,36 @@ https://doi.org/10.1038/s41598-022-11210-y
 On Linux and Windows the project can be used by running the following commands to clone the repository and install the required dependencies.
 
 Either with `anaconda` or `miniconda` installed
-```
+```shell
 git clone https://github.com/dslaborg/sumo.git
 cd sumo
 conda env create --file environment.yaml
 conda activate sumo
 ```
 or using `pip`
+```shell
+git clone https://github.com/dslaborg/sumo.git
+cd sumo
+pip install -r requirements.txt
 ```
+
+`pip` with virtualenv
+```shell
+# enter the project root
 git clone https://github.com/dslaborg/sumo.git
 cd sumo
+
+# Create the virtual env with pip
+virtualenv venv --python=python3.9.10
```

Comment:
```suggestion
venv venv
```

As far as I know, since python 3.3 `venv` is already natively included and the prefered way. `virtualenv` only works with python 2 I think?

Reply:
Okay, LGTM